bin/miden-cli/src/lib.rs

```diff
@@ -43,6 +43,9 @@ const CLIENT_CONFIG_FILE_NAME: &str = "miden-client.toml";
 /// Client binary name.
 pub const CLIENT_BINARY_NAME: &str = "miden";
 
+/// The number of blocks that are considered old enough to discard pending transactions.
```

Comment:
```suggestion
/// Number of blocks that must elapse after a transaction’s reference block before it is marked stale and discarded.
```


I'd also change the description for similar consts/functions in the other files.

Reply:
Should we also change the const name to `TX_GRACEFUL_BLOCK_DELTA`? I think the `CLI` is a bit ambiguous since it's on the CLI crate anyway.